### DISCUSSION TITLE
Apple TV Support!

### DISCUSSION THREAD

**Discussion by jahangirjr** [f1multiviewer/issue-tracker#281] [Feature requests] [answered]
I'm aware we can airplay. but if we can play directly from Apple TV we can reduce network issues and buffering.

Lets get some Apple TV love

**Comment by f1multiviewer** [comment, marked as answer]
Actually you *can't* airplay live content because of DRM on live/instant-vod content. I think the best alternative for MultiViewer for F1 that runs natively on Apple TV is @noahfetz's https://github.com/NoahFetz/F1AppleTV, which can also run multi-feed playback. I don't have any experience myself with native applications, so that's out of scope for me, but I've said countless times that if other developers want to take on a project like that, I'd be happy to assist them in whatever way (and have helped others in our #technical channel on Discord)

**Comment by ely105** [comment]
With all the DRM issues on the feeds it would be great if the AppleTV could be a slave device that is controlled by f1mv but is able to receive streams directly to allow for DRM.  It's quite an undertaking though so I don't imagine it'd be easy to pull off.

**Reply by NoahFetz** [reply]
To pull this off, you'd need an app on the device itself anyway, and at this point you can just embedd all the other logic as well

**Reply by f1multiviewer** [reply]
Yeah you'd have to re-write everything from MultiViewer specifically for Apple TV. That's not something I can do alone. Happy to help others and allow apps to communicate with MultiViewer over the network to sync, but this would be a pretty huge challenge to get up and running.
